src/app/app.component.ts

```diff
@@ -17,7 +16,7 @@ export class AppComponent {
     this.currentSearchData$ = this.store.select('CurrentSearchReducer');
```

Comment:
Anyway, as this discussion it's not related to any change introduced by this PR, I suggest to do the merge (when I'd push another fix commit relating to oher comments) and discuss this besides this PR. For the sake of being a little more agile and move on. What do you think?